src/components/addresses/AddressGroupForm.js

```diff
@@ -3,23 +3,28 @@ import { Formik, Form } from "formik";
 import Input from "components/forms/Input";
 import SubmitSpinnerButton from "components/forms/SubmitSpinnerButton";
 import ErrorMessage from "components/ErrorMessage";
+import isNumeric from "validator/lib/isNumeric";
 
 const defaultInitialValues = {
-  percent: 100,
+  percent: "100",
   label: "",
   address1: "",
   coin: "BTC",
   userID: ""
 };
-const validate = ({ label, address1 }) => {
+const validate = ({ label, address1, percent }) => {
   const errors = {};
   const reqMsg = "This field is required";
   if (!label) errors.label = reqMsg;
   if (!address1) errors.address1 = reqMsg;
+  if (!isNumeric(percent.toString()))
+    errors.percent = "percent must be a number";
+  if (Number(percent) < 0 || Number(percent) > 100)
+    errors.percent = "percent must be between 0 and 100";
```

Comment:
Should these have capitals?
```suggestion
    errors.percent = "Percent must be a number";
  if (Number(percent) < 0 || Number(percent) > 100)
    errors.percent = "Percent must be between 0 and 100";
```